### PR TITLE
added style.css and changed webpage

### PR DESCRIPTION
**Pull request by gurtatiLND**
**Summary:**
This PR includes the following changes to the project:

1. Added a new stylesheet (`styles.css`).
2. Filled out the `<section>` element on `webpage.html`.
3. Added a background picture to the webpage.

- Please review the changes, especially the new styles and the content added to the `<section>` element.
- Let me know if there are any improvements or additional changes required.

Thank you!

### CODE REVIEW

**Review by sofer — APPROVED** [review]
Looks OK. You may want to look at using a .gitignore file to deal with .DS_Store files, that you don't really need in your committed code.

File: users/gurtatiLND/website/styles.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    color:rgb(246, 244, 241);
+    background-image: url(./assets/home_background.jpg);
+    background-color:rgb(52, 100, 100);
+    background-position: center;
+    height: 90vh;
+    /*min-height: 80rem;
+    max-height: 120rem;*/
+    position: relative;
+}
+.home_content {
```
Comment (CringleySDays) [inline users/gurtatiLND/website/styles.css:25]: Follow the kebab-case convention when naming identifiers. Fe 

.home-content {
 ...
}

File: users/gurtatiLND/website/webpage.html [hunk 2]
```diff
@@ -11,14 +12,45 @@
...
+            <section id="projects" class="projects sec-pad">
+                <div class="main_container">
+                    <h2 class="heading projects">PROJECTS</h2>
+                    <div class="projects_content">
+                        <p>My projects:</p>
+                    </div>
+                </div>
+            </section>
+            <section id="contact" class="contact sec-pad">
+                <div class="main_container">
```
Comment (CringleySDays) [inline users/gurtatiLND/website/webpage.html:43]: kebab-case please

File: users/gurtatiLND/website/webpage.html [hunk 3]
```diff
@@ -11,14 +12,45 @@
...
+                    <h2 class="heading contact">CONTACT</h2>
```
Comment (CringleySDays) [inline users/gurtatiLND/website/webpage.html:44]: kebab-case please

**Review by CringleySDays — CHANGES_REQUESTED** [review]

**Comment by CringleySDays** [comment]
I'm not too sure what the .DS_STORE files but if it will contain sensitive info like Database keys, user data storage (I'd recommend going with SQL databases), No-SQL will work but it depends how what data **structures** you'll be storing

Research .gitignore files on how to deal with it as sofer mentioned @gurtatiLND 

Looking forward to the main page being customised, well done!

File: users/gurtatiLND/website/webpage.html [hunk 4]
```diff
@@ -11,14 +12,45 @@
...
+                    <h1 class="heading_home">HI, I'M TATIANA GUROVA</h1>
+                    <div class="home_info">
+                        <p class="text_home">Frontend Developer</p>
+                    </div>
+                    <div class="home_link">
+                        <a href="">Projects</a>
+                    </div>
+                </div>
+            </section>
+            <section id="about" class="about sec-pad">
```
[the remark this answers is not in the document]
Comment (gurtatiLND) [reply users/gurtatiLND/website/webpage.html:26]: Do you mean at this part **class="about sec-pad"**? 
But this section has styles from both the "about" and "sec-pad" classes.

File: users/gurtatiLND/website/webpage.html [hunk 5]
```diff
@@ -11,14 +12,45 @@
...
+                <div class="main_container">
+                    <h2 class="heading about">ABOUT</h2>
```
[the remark this answers is not in the document]
Comment (gurtatiLND) [reply users/gurtatiLND/website/webpage.html:28]: In this case the same: this section has styles from both the "heading" and "about" classes.

File: users/gurtatiLND/website/webpage.html [hunk 6]
```diff
@@ -11,14 +12,45 @@
...
+                    <div class="about_content">
+                        <p>How I became a frontend dev</p>
+                    </div>
+                </div>
+            </section>
+            <section id="projects" class="projects sec-pad">
```
[the remark this answers is not in the document]
Comment (gurtatiLND) [reply users/gurtatiLND/website/webpage.html:34]: In this section has styles from both the "projects" and "sec-pad" classes.

File: users/gurtatiLND/website/webpage.html [hunk 7]
```diff
@@ -11,14 +12,45 @@
...
+                <div class="main_container">
+                    <h2 class="heading projects">PROJECTS</h2>
```
[the remark this answers is not in the document]
Comment (gurtatiLND) [reply users/gurtatiLND/website/webpage.html:36]: This section has styles from the "heading" and "projects" classes.

File: users/gurtatiLND/website/webpage.html [hunk 8]
```diff
@@ -11,14 +12,45 @@
...
+                    <div class="projects_content">
+                        <p>My projects:</p>
+                    </div>
+                </div>
+            </section>
+            <section id="contact" class="contact sec-pad">
```
[the remark this answers is not in the document]
Comment (gurtatiLND) [reply users/gurtatiLND/website/webpage.html:42]: This section has styles from the "contact" and "sec-pad" classes.

File: users/gurtatiLND/website/webpage.html (hunk 3 above)
Comment (gurtatiLND) [reply users/gurtatiLND/website/webpage.html:44]: This section has styles from the "heading" and "contact" classes.

File: users/gurtatiLND/website/webpage.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (CringleySDays) [reply users/gurtatiLND/website/webpage.html:42]: yup should be named class="contact-sec-pad" for consistency

[the remark this answers is not in the document]
Comment (CringleySDays) [reply users/gurtatiLND/website/webpage.html:42]: Wait one second, it has styles from both of them? There's a way to solve that

[the remark this answers is not in the document]
Comment (gurtatiLND) [reply users/gurtatiLND/website/webpage.html:42]: Yes

[the remark this answers is not in the document]
Comment (CringleySDays) [reply users/gurtatiLND/website/webpage.html:42]: Nvm, I'm thinking about SASS which solves that issue, my bad

**Review by CringleySDays — APPROVED** [review]

File: users/gurtatiLND/website/webpage.html (hunk 8 above)
[the remark this answers is not in the document]
Comment (gurtatiLND) [reply users/gurtatiLND/website/webpage.html:42]: I applied two classes to one element and found this solution to be the best.
https://www.geeksforgeeks.org/how-to-apply-two-css-classes-to-a-single-element/

[the remark this answers is not in the document]
Comment (CringleySDays) [reply users/gurtatiLND/website/webpage.html:42]: In SASS (which you can use) which was a simple method but CSS doesn't have that syntax unfortunately

there's a syntax where you can do it like

.new-class-name {
 @extend contact;
 @extend sec-pad;

 // You can add additional properties unique to this //
}

[the remark this answers is not in the document]
Comment (CringleySDays) [reply users/gurtatiLND/website/webpage.html:42]: Everything looks good to me, you're happy for me to PR?

**Pull request merged**